onlinejudge/service/yukicoder.py

```diff
@@ -26,10 +26,10 @@ def get_url_of_login_page(self):
 
     def is_logged_in(self, *, session: Optional[requests.Session] = None) -> bool:
         session = session or utils.get_default_session()
-        url = 'https://yukicoder.me/auth/github'
+        url = 'https://yukicoder.me'
         resp = utils.request('GET', url, session=session, allow_redirects=False)
```

Comment:
```suggestion
        resp = utils.request('GET', url, session=session)
```

nitpick: `allow_redirects=False` はもういらなくなってるから消しておいてほしい